class.jetpack.php

```diff
@@ -2366,6 +2366,11 @@ function admin_init() {
 			add_action( 'load-index.php', array( $this, 'prepare_manage_jetpack_notice' ) );
 		}
 
+		// If the plugin has just been disconnected from WP.com, show the survey notice
+		if ( isset( $_GET['disconnected'] ) && 'true' === $_GET['disconnected'] ) {
+			add_action( 'admin_notices', array( $this, 'disconnect_survey_notice' ) );
```

Comment:
We have a `jetpack_notices` hook :) It will put the notice in a more appropriate place.  

Great idea BTW